data/wanderer/wanderers start.txt

```diff
@@ -17,7 +17,7 @@ mission "First Contact: Wanderer (Before Hai)"
 	on offer
 		conversation
 			`All the other worlds belonging to this species have appeared unwilling to let you land, but here you find a slightly warmer welcome. A cloud of small, highly maneuverable patrol ships takes off from the planet as you approach and herds you onto one particular landing pad where, after a few minutes, several of the aliens approach your ship.`
-			`	This species is birdlike, but with bodies nearly as large as yours, and with wings that tower above you. They come swooping down from the sky, circle your ship to inspect it, and then land near the hatch and wait for you to open it. They do not appear to be armed, but their talons are three or four inches long.`
+			`	This species is birdlike, but with bodies nearly as large as yours, and with wings that tower above you. They come swooping down from the sky, circle your ship to inspect it, and then land near the hatch and wait for you to open it. They do not appear to be armed, but their talons are substantial.`
```

Comment:
Why not retain a measurement and say something like “eight or nine centimeters long” both here and below.

Reply:
SI convention which I think is followed here usually prefers millimetres over centimetres and "twenty or thirty millimetres" long didn't sound natural to me, happy to defer to you if you'd prefer that wording though?

Reply:
Well it's not so much that the game follows "SI convention" than it just uses metric measurements, and just as "three or four inches long" would be used in normal speech, I don't see why "eight or nine centimeters long" wouldn't.

Reply:
Places that, uh, drink the metric kool-aid thoroughly like Japan rarely use centimetres, which are mostly a phenomenon of anglophone countries picking the closest amount to an inch. That said, just let me know what wording you want.

Reply:
I'd say just use centimeters. We use American English spellings, so not like we don't already break from more "proper" standards.

Reply:
All English spelling systems are equally broken anyway :)

Alright, wilco

Reply:
Speaking from personal experience, there's definitely usage of centimetres outside the context of translating inches in at least some metric-heavy countries, especially in situations like the one in the quote where it is a quick estimate.

Reply:
fde3f78e